web/experimental/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+    <head>
+        <meta http-equiv="refresh" content="0; url=embed-new.html" />
+    </head>
+    <body>
+        Redirecting to <a href="embed-new.html">embed-new.html</a>
```

Comment:
I'd probably point people to `new_embeddings_demo.html` instead.

Reply:
Done,